src/quo2/components/links/link_preview/view.cljs

```diff
@@ -54,7 +54,9 @@
   [logo]
   [rn/image
    {:accessibility-label :logo
-    :source              logo
+    :source              (if (string? logo)
+                           {:uri logo}
+                           logo)
```

Comment:
I am curious why do we need this condition?

Reply:
This condition is needed because sometimes the source of the image is a data URI, which is a string. Other times it's the result of `resources/get-mock-image`, which is number uniquely identifying the resource. The check is needed to remove the warning.